### DISCUSSION TITLE
Azure Active Directory SAML auth with Fastify

### DISCUSSION THREAD

**Discussion by mnemosdev** [fastify/fastify#2905] [Q&A]
Hello everyone, I would like to know whether there's already some pluggable module to auth my app with SAML

Thanks to anyone who might help or read!

**Comment by mcollina** [comment]
I don't think we have ever developed a SAML plugin. However the company I work for have used this https://github.com/nearform/titus/blob/master/packages/titus-backend/lib/plugins/auth/azure-ad/index.js.

**Reply by mnemosdev** [reply]
Thanks Matteo! I'll check it out
